### ISSUE TITLE
No Amiga support?

### ISSUE THREAD

**Issue by mvirkkunen** [sonountaleban/amigacoin#1]
I couldn't find any information on how to build and run this on an Amiga computer. How can a project called Amigacoin not run on Amiga?

**Comment by sonountaleban** [comment]
Yes, you're completely right and it's a big lack. However I thought that may be a way to support/advertise our platform. Maybe I'll work on it in the future. Thanks.
